bootstraptest/test_yjit.rb

```diff
@@ -2735,3 +2735,13 @@ def foo(s)
   foo(s) rescue :ok
   foo(s) rescue :ok
 }
+
+# Ruby array varg
```

Comment:
Nitpick but comment should be more specific that File.join is a C method that takes vararg array 😅